src/main/java/gameframework/drawing/DrawableImage.java

```diff
@@ -17,7 +17,14 @@ public DrawableImage(URL imageUrl, GameCanvas gameCanvas) {
 		}
 		handleImage(imageUrl);
 	}
-
+	
```

Comment:
this comment has nothing to do with this pull request. Please open a separate one.